chef/data_bags/crowbar/bc-template-provisioner.json

```diff
@@ -79,6 +79,14 @@
       "windows": {
         "admin_password": "crowbar"
       },
+      "suse": {
+        "autoyast": {
+          "packages": {
+            "suse-11_3": [ ],
+            "suse-12_0": [ ]
```

Comment:
The dot was replaced with an underscore for a reason. I gave a very brief explanation in the commit message: The i18n library cannot handle names like `suse-11.3`. Best option was to replace it with an underscore.